### ISSUE TITLE
iPad: Can't run notebook cell with Keyboard

### ISSUE THREAD

**Issue by joaomoreno** [microsoft/vscode#139455]
1. Open a notebook, focus a cell
2. Press any combination of <kbd>Cmd</kbd>, <kbd>Ctrl</kbd>, <kbd>Shift</kbd>, <kbd>Alt</kbd>, <kbd>Enter</kbd>. 🐛  Nothing seems to make the cell run.

🐛 `Run Cell` doesn't appear in the command palette, so I can't figure out the keyboard action from there.
🐛 It also doesn't appear in the cell's `...` menu:

![5A20D050-F241-48B2-AB32-0E64201F2726](https://user-images.githubusercontent.com/22350/146666183-a2ee2135-364e-4f50-9632-a3a370e93e4f.jpeg)

I could only find out the actual shortcut <kbd>Ctrl Enter</kbd> by going into Keyboard Shortcut Preferences.

**Comment by rebornix** [comment]
@joaomoreno I tried to reproduce but didn't succeed but I did notice something interesting, can you help confirm:

* Touch/Click on a notebook file from File Explorer
* The notebook is opened, but the editor is not focused. You can tell from the color of the cell focus border, if it's focused, it's blue-ish otherwise it's gray
  * I found that if I click on a text file from File Explorer, the Monaco editor is also not focused. I wonder if it's a regression of the List View
 * Since the editor/cell is not focused, Ctrl+Enter will not execute.
 * You can click/touch on the cell, make sure the focus border is blue, and then lastly run Ctrl+Enter, it should execute or show a kernel picker.

**Comment by joaomoreno** [comment]
Yes, I can repro that the first text editor within a notebook editor isn't focused when opening the notebook from the Explorer.

But I can still repro my issue. I have a CH-DE keyboard layout, which I'm suspecting is the culprit.

The only combination which works is <kbd>Shift Enter</kbd>, but that creates a new cell.
